e2e/libs/keywords/volume_keywords.py

```diff
@@ -1,34 +1,47 @@
-from pathlib import Path
-from volume import Volume
-from utility import Utility
-from node import Node
+from utility.utility import get_test_case_namespace, generate_volume_name
+from utility.utility import get_node, list_nodes
+from utility.utility import get_test_pod_running_node, get_test_pod_not_running_node
+from robot.libraries.BuiltIn import BuiltIn
 from node_exec import NodeExec
+from volume import Volume
+import logging
 
 class volume_keywords:
 
-    ROBOT_LIBRARY_SCOPE = 'TEST'
-
     def __init__(self):
-        Utility().init_k8s_api_client()
-
-
-    def set_test_name(self, test_name):
-        self.namespace = test_name.lower().replace(' ', '-')
-        self.node_exec = NodeExec(self.namespace)
+        #TODO
+        #test_name = BuiltIn().get_variable_value("${TEST NAME}")
+        self.node_exec = NodeExec("default")
         self.volume = Volume(self.node_exec)
 
 
     def create_volume(self, size, replica_count):
-        print('create_volume')
-        volume_name = Utility().generate_volume_name()
+        volume_name = generate_volume_name()
         self.volume.create(volume_name, size, replica_count)
+        logging.info(f'==> create volume {volume_name}')
```

Comment:
Temporarily not modify this since the original purpose is to have useful runtime information. What's the correct way to achieve this is still unknown. `logging.warn` doesn't look right either since it will generate extra messages in robot test report.

Reply:
Generating debug message in robot report is ok if logs are reasonable.

If don't know now, again don't spend time on it now but I believe logs somehow will be used like u want to print some debug info.